packages/story-editor/src/components/videoTrim/slider.js

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { useRef, useCallback } from '@web-stories-wp/react';
+import { useKeyDownEffect } from '@web-stories-wp/design-system';
+
+const Thumb = styled.button`
+  position: absolute;
+  padding: 0;
+`;
+
+function Slider({
+  railWidth,
+  min = 0,
+  max = 10000,
```

Comment:
Same Q here -- will this be the max video length in ms?

Reply:
This default is also a bit weird. Should be removed I guess.

Reply:
Done in 03bb4c1798.